### PR TITLE
Feature/add structure

### PR DESCRIPTION
**Pull request by kangyeon0728**
made Home page, About page, Culture page, Vision page
Home page is made of HeroSection, Cards, Footer components
About page is made of AboutM6, Footer components
Culture page is made of Member, Footer components
Vision page is made of Ourvision, Footer components.
Vision page and specific design of Culture page is not done yet

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			</section>
+
+			<section class="social-media">
+				<div class="social-media-wrap">
+					<div class="footer-logo">
+						<Link to="/" className="navbar-logo">
+							<img src="images/M6_Logo white png.png" alt="" className="logo" />
+						</Link>
+					</div>
+					<small class="website-rights">M6 © 2020</small>
```
Comment (sanjuruk) [inline src/components/Footer.js:50]: 2020? It is 2021!

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import "./Footer.css";
+import Button from "./Button";
+import { Link } from "react-router-dom";
+import ButtonMailto from "./ButtonMailto";
```
Comment (sanjuruk) [inline src/components/Footer.js:5]: Build fails with this line

**Review by sanjuruk — CHANGES_REQUESTED** [review]
See the comments

File: src/components/Footer.js (hunk 1 above)
Comment (kangyeon0728) [reply src/components/Footer.js]: got it!

File: src/components/Footer.js (hunk 2 above)
Comment (kangyeon0728) [reply src/components/Footer.js]: figured it out

**Pull request merged**